meta-zarhus-distro/recipes-tools/zarhus-memtools/zarhus-memtools_1.0.0.bb

```diff
@@ -0,0 +1,28 @@
+SUMMARY = "Tools for measuring and analyzing memory usage of a process"
+HOMEPAGE = "https://docs.zarhus.com"
+
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+SRC_URI = "\
+    file://mem_usage \
+    file://mem_test \
+    file://plot_mem_usage.py \
+    "
+
+S = "${WORKDIR}"
+
+do_install() {
+    install -d ${D}${bindir}
+    install -m 0755 ${S}/mem_test ${D}${bindir}
+    install -m 0755 ${S}/plot_mem_usage.py ${D}${bindir}
+    install -m 0755 ${S}/mem_usage ${D}${bindir}
+}
+
+FILES:${PN} += " \
+    ${bindir}/mem_usage \
+    ${bindir}/mem_test \
+    ${bindir}/plot_mem_usage.py \
+    "
+
+RDEPENDS:${PN} += "bash python3 python3-matplotlib python3-numpy"
```

Comment:
Add heaptrack. If we ever include this layer and want memtools in prod image then we'd only need to include this recipe.
And you can remove those rdpeneds from packagegroup (heaptrack/python3-* or maybe even python3 (though it could be useful outside this recipe so it can stay))  as long as you include zarhus-memtools in packagegroup

Reply:
There is also bash rdepends. Does it really use bash or is sh enough? Maybe `sh` is enough? If it is then also fix this line in scripts: `#!/bin/bash`

Reply:
Memtools don't require heaptrack to run since I removed the script that uses it. The users may want to only use the `mem_test` script.

Reply:
You are right my bad.

Reply:
```
ERROR: Nothing RPROVIDES 'sh'
```

```
ls -l /bin/sh
lrwxrwxrwx 1 root root 18 Mar  9  2018 /bin/sh -> /usr/bin/bash.bash
```

We have bash in the image

Reply:
I know, just asked in case we ever want to use this recipe in image that doesn't have bash. Not sure if in case of sh we need to add anything to rdepends.